ui/components/NFTS_update/Filters/__tests__/FilterList.test.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import FilterList from "../FilterList"
+import { STARS_GREY_URL } from "../../../Shared/SharedToggleItem"
+
+const EMPTY_MESSAGE = "Empty list"
+const FILTER_LIST = [
+  {
+    id: "1",
+    name: "Item 1",
+    thumbnailURL: "./images/item_1.svg",
+    isEnabled: false,
+  },
+  {
+    id: "2",
+    name: "Item 2",
+    thumbnailURL: "./images/item_2.svg",
+    isEnabled: true,
+  },
+  {
+    id: "3",
+    name: "Item 3",
+    isEnabled: true,
+  },
+]
+const onChange = jest.fn()
+
+describe("FilterList", () => {
+  test("should render a component", () => {
+    const ui = render(
+      <FilterList
+        filters={FILTER_LIST}
+        onChange={onChange}
+        emptyMessage={EMPTY_MESSAGE}
+      />
+    )
+    const list = ui.getByTestId("nft_filters_list")
+
+    expect(list).toBeInTheDocument()
+  })
+
+  test("should display an empty message", () => {
+    const ui = render(
+      <FilterList
+        filters={[]}
+        onChange={onChange}
+        emptyMessage={EMPTY_MESSAGE}
+      />
+    )
+    const emptyMessage = ui.queryByText(EMPTY_MESSAGE)
+
+    expect(emptyMessage).toBeInTheDocument()
+  })
+
+  test("should render two items", () => {
```

Comment:
```suggestion
  test("should render three items", () => {
```
Let's replace the first test with this one and add a snapshot on `ui.container` to prevent visual regressions

Reply:
Are we sure about it? I remember that we made a decision not to do snapshot tests.  @jagodarybacka @0xDaedalus

Reply:
Yeah, seems like we decided to skip snapshot testing https://github.com/tahowallet/extension/issues/2310

Reply:
Ah yeah, I agree, maybe snapshots for shared components would be a good idea. At least we can try. Let's discuss it at some point.